portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -85,10 +89,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String email = request.getParameter("email").trim();
     if (email.isEmpty()) email = "@";
 
+    Document doc = Document.newBuilder().setContent(comment).setType(Document.Type.PLAIN_TEXT).build();
```

Comment:
You could probably extract this out into a helper function so as not to bloat this one

Reply:
Good point.